cmake/multi_image.cmake

```diff
@@ -317,51 +321,65 @@ function(add_child_image_from_source)
     #       files must be used instead of the child image default configs.
     #       The append a child image default config, place the additional settings
     #       in `child_image/<ACI_NAME>.conf`.
-    set(ACI_CONF_DIR ${APPLICATION_CONFIG_DIR}/child_image)
-    set(ACI_NAME_CONF_DIR ${APPLICATION_CONFIG_DIR}/child_image/${ACI_NAME})
-    if (NOT ${ACI_NAME}_CONF_FILE)
-      ncs_file(CONF_FILES ${ACI_NAME_CONF_DIR}
-        BOARD ${ACI_BOARD}
-        # Child image always uses the same revision as parent board.
-        BOARD_REVISION ${BOARD_REVISION}
-        KCONF ${ACI_NAME}_CONF_FILE
-        DTS ${ACI_NAME}_DTC_OVERLAY_FILE
-        BUILD ${CONF_FILE_BUILD_TYPE}
-        )
-      # Place the result in the CMake cache and remove local scoped variable.
-      foreach(file CONF_FILE DTC_OVERLAY_FILE)
-        if(DEFINED ${ACI_NAME}_${file})
-          set(${ACI_NAME}_${file} ${${ACI_NAME}_${file}} CACHE STRING
-            "Default ${ACI_NAME} configuration file" FORCE
-            )
-	  set(${ACI_NAME}_${file})
+    zephyr_get(COMMON_CHILD_IMAGE_CONFIG_DIR)
+    string(CONFIGURE "${COMMON_CHILD_IMAGE_CONFIG_DIR}" COMMON_CHILD_IMAGE_CONFIG_DIR)
+    foreach(config_dir ${APPLICATION_CONFIG_DIR} ${COMMON_CHILD_IMAGE_CONFIG_DIR} )
+      set(ACI_CONF_DIR ${config_dir}/child_image)
+      set(ACI_NAME_CONF_DIR ${config_dir}/child_image/${ACI_NAME})
+      if (NOT ${ACI_NAME}_CONF_FILE)
+        ncs_file(CONF_FILES ${ACI_NAME_CONF_DIR}
+          BOARD ${ACI_BOARD}
+          # Child image always uses the same revision as parent board.
+          BOARD_REVISION ${BOARD_REVISION}
+          KCONF ${ACI_NAME}_CONF_FILE
+          DTS ${ACI_NAME}_DTC_OVERLAY_FILE
+          BUILD ${CONF_FILE_BUILD_TYPE}
+          SUFFIX ${${ACI_NAME}_FILE_SUFFIX}
```

Comment:
Not correct, needs fixing before merge https://github.com/gmarull/sdk-nrf/pull/17#discussion_r1575882251

Reply:
@tejlmand